data/autoyast_sle15/mini_s390x.xml

```diff
@@ -41,6 +67,15 @@
             <package>openssh</package>
         </packages>
     </software>
+    <services-manager>
```

Comment:
I guess the issue is firewall enabled on new build but ssh port hasn't been enabled for firewall, so no problem for firewall section, not sure we need the service section, please double confirm.

Reply:
These are the VRs:
https://openqa.suse.de/tests/17473763# (with <services> only)
https://openqa.suse.de/tests/17473824# (with <firewalld> only)
All failed with reconnecting the system.

Reply:
I think you misunderstanding, I mean the service section may not needed, could you remove it and run again to verify? ( reserve the firewall setting to add port for ssh)

Reply:
As I said above, this is the VR that reserve the firewall setting to add port for ssh only: https://openqa.suse.de/tests/17473824#

Reply:
Ok, so it seems there is something changed in latest build : firewall seems enabled by default and ssh hasn't been enabled.